packages/protvista-utils/src/Region.js

```diff
@@ -8,11 +8,11 @@ export default class Region {
 
   encode(full = false) {
     return this.segments
-      .map(({ start, end }) => {
-        if (full) return `${start}:${end}`;
+      .map(({ start, end, color }) => {
+        if (full) return `${start}:${end}:${color ?? "Default"}`;
```

Comment:
Could color always be optional?
```suggestion
        if (full) return `${start}:${end}${color ? `:${color}` : ""}`;
```